test/boolean_complex_test.cpp

```diff
@@ -1030,4 +1031,160 @@ TEST(BooleanComplex, HullMask) {
   MeshGL mesh = ret.GetMeshGL();
 }
 
+TEST(BooleanComplex, SimpleOffset) {
+  std::string file = __FILE__;
+  std::string dir = file.substr(0, file.rfind('/'));
+  MeshGL seeds = ImportMesh(dir + "/models/" + "Generic_Twin_91.1.t0.glb");
+  EXPECT_TRUE(seeds.NumTri() > 10);
+  EXPECT_TRUE(seeds.NumVert() > 10);
+  // Find unique edges
+  std::set<std::pair<int, int>> edges;
+  for (size_t i = 0; i < seeds.NumTri(); i++) {
+    int eind[3];
+    for (int j = 0; j < 3; j++) eind[j] = seeds.triVerts[i*3+j];
+    int e11, e12, e21, e22, e31, e32;
+    e11 = (eind[0] < eind[1]) ? eind[0] : eind[1];
+    e12 = (eind[1] < eind[0]) ? eind[0] : eind[1];
+    e21 = (eind[1] < eind[2]) ? eind[1] : eind[2];
+    e22 = (eind[2] < eind[1]) ? eind[1] : eind[2];
+    e31 = (eind[2] < eind[0]) ? eind[2] : eind[0];
+    e32 = (eind[0] < eind[2]) ? eind[2] : eind[0];
+    edges.insert(std::make_pair(e11, e12));
+    edges.insert(std::make_pair(e21, e22));
+    edges.insert(std::make_pair(e31, e32));
```

Comment:
Just a nit, but this is way more complicated than it needs to be. Have you considered something like:
```
std::vector<std::pair<int, int>> edges;
for (size_t i = 0; i < seeds.NumTri(); i++) {
  int k[3]={1,2,0};
  for(const int j:{0,1,2}){
    int v1=seeds.triVerts[i*3+j];
    int v2=seeds.triVerts[i*3+k[j]];
    if(v2>v1) edges.push_back(std::make_pair(v1, v2));
  }
}
```

Reply:
Ah - I hadn't boiled it down that far.  Updated.